workers/unity/Packages/com.improbable.gdk.core/Subscriptions/Systems/RequireLifecycleSystem.cs

```diff
@@ -13,6 +13,7 @@ public class RequireLifecycleSystem : ComponentSystem
         private CommandCallbackSystem commandCallbackSystem;
         private ComponentCallbackSystem componentCallbackSystem;
         private ComponentConstraintsCallbackSystem componentConstraintsCallbackSystem;
+        private WorkerFlagCallbackSystem workerFlagCallbackSystem;
 
         // todo make this order the behaviours somehow
```

Comment:
link ticket

Reply:
I think its this one - https://improbableio.atlassian.net/browse/UTY-877

@samiwh to confirm